programs/sll_remove_dups.py

```diff
@@ -0,0 +1,48 @@
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+        self.head = None
+
+    def display(self):
+        if self.head is None:
+            print("Linked List is empty")
+        else:
+            temp = self.head
+            while temp:
+                print(temp.data, end="  ")
+                if temp.next:
+                    print("--->", end=" ")
+                temp = temp.next
+            print()
+
+    def RemDups(self):
```

Comment:
This method (your answer) should always be a standalone function that receives a head. That is the format of interviews.

Also, declare method the python way with `snake_casing`

```suggestion
def remove_dups(head:Node):
```